src/openzaak/components/catalogi/models/besluittype.py

```diff
@@ -131,3 +137,12 @@ def __str__(self):
         if self.concept:
             representation = "{} (CONCEPT)".format(representation)
         return representation
+
+    def save(self, *args, **kwargs):
+        if not self.pk:
+            transaction.on_commit(AutorisatieSpec.sync)
```

Comment:
This only makes sense if the `save` is wrapped in a `transaction.atomic` right?

Reply:
Or an outer block of code is wrapped in atomic transaction, which the admin change views are

Reply:
Earlier a saw the save wrapped in a `transaction.atomic`. The behaviour is then slightly different, and the sync is called AFTER the save. Without wrapping the save, it's called BEFORE the save.

I'm not sure if this matters though.

Reply:
Code updated!